modules/tile-converter/src/i3s-converter/helpers/geometry-converter.js

```diff
@@ -396,6 +384,7 @@ function flattenColors(colorsAttribute, indices) {
   const components = colorsAttribute?.components || VALUES_PER_COLOR_ELEMENT;
   const newColors = new Uint8Array(indices.length * components);
   if (!colorsAttribute) {
+    newColors.fill(255);
```

Comment:
Explain in comment that this is intended to be "all white" by default

Reply:
Done